.github/workflows/validate-pull-request.yml

```diff
@@ -0,0 +1,9 @@
+name: Validate Pull Request
+
+on:
+  pull_request:
+
+jobs:
+  validate_pull_request:
+    name: Call validate workflow
+    uses: Lombiq/GitHub-Actions/.github/workflows/validate-pull-request.yml@dev
```

Comment:
This should remain the same as before.

Reply:
But it should execute the validate-pull-request not the validate-submodule-request :

> validate-submodule-pull-request is now validate-pull-request, and its content also changed, but it still only executes the validate-submodule-pull-request workflow.

Reply:
>but it still only executes the validate-submodule-pull-request workflow.
Also please see Zoli pointing out as well [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/301#issuecomment-1412947826).

Reply:
okay, I just saw the comment. but the sentence seemed like executing the validate-submodule-pull-request is not what is intended to do .. nvm

Reply:
done

Reply:
I don't see that it was changed, did you push?

Reply:
yes